packages/flutter/lib/src/material/navigation_bar.dart

```diff
@@ -0,0 +1,1144 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import 'color_scheme.dart';
+import 'elevation_overlay.dart';
+import 'ink_well.dart';
+import 'material.dart';
+import 'material_localizations.dart';
+import 'theme.dart';
+import 'tooltip.dart';
+
+/// Material 3 Navigation Bar component.
+///
+/// This widget holds a collection destinations (usually
+/// [NavigationBarDestination]s).
+///
+/// Usage:
+/// ```dart
+/// Scaffold(
+///   bottomNavigationBar: NavigationBar(
+///     onTap: (i) => setState(() => _currentPageIndex = i),
+///     selectedIndex: _currentPageIndex,
+///     destinations: [
+///       NavigationBarDestination(
+///         icon: Icon(Icons.explore),
+///         label: 'Explore',
+///       ),
+///       NavigationBarDestination(
+///         icon: Icon(Icons.commute),
+///         label: 'Commute',
+///       ),
+///       NavigationBarDestination(
+///         selectedIcon: Icon(Icons.bookmark),
+///         icon: Icon(Icons.bookmark_border),
+///         label: 'Saved',
+///       ),
+///     ],
+///   ),
+/// ),
+/// ```
+class NavigationBar extends StatelessWidget {
+  /// Creates a Material 3 Navigation Bar component.
+  const NavigationBar({
+    Key? key,
+    this.animationDuration = const Duration(milliseconds: 500),
+    this.selectedIndex = 0,
+    required this.destinations,
+    this.onDestinationSelected,
+    this.elevation = 8,
```

Comment:
Default elevation should be 0, or perhaps not even customizable since this component is not supposed to have a shadow now